docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/tag-template-guide/index.md

```diff
@@ -36,7 +38,7 @@ Finally, the tag bundles a **command** from the settings in the tag, and sends i
 
 ### Settings Configuration
 
-The Tag template requires a [Snowplow v3 Settings](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/settings-variable-guide/index.md) Variable template to be configured which can be referenced within the Tag. This settings template contains the information required for the Tag to appropriate initialize the tracker.
+The Tag template requires a [Snowplow v3 Settings](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-settings-variable/index.md) Variable template to be configured which can be referenced within the Tag. This settings template contains the information required for the Tag to appropriate initialize the tracker.
```

Comment:
s/appropriate/appropriately/

Reply:
For the future, should we have more detail here on which Tags require actually referencing the Settings? Especially if you're basing everything off the `gtm.js` event filtering off e.g. URL (poor practice, but pretty common), it can be important to do it in multiple Tags and potentially keep them in sync but we don't seem to have any guidance on that, or explain the behaviour when multiple Tags all have "Tracker Initialisation" configured. (This is particularly unclear if you have "overrides" configured for some Tags but not others)